posts/2024-10-31-project-goals-oct-update.md

```diff
@@ -0,0 +1,254 @@
+    <div style="flex: initial;"><progress value="15" max="37"></progress>
+</div>
+</div>
+
+The biggest elements of our goal are solving the ["send bound" problem][sbp] via [return-type notation][RTN] (RTN) and adding support for async closures. This month we made progress towards both. For RTN, @compiler-errors extended the return-type notation landed support for using RTN in self-types like `where Self::method(): Send`. He also authored a [blog post with a call for testing](https://blog.rust-lang.org/inside-rust/2024/09/26/rtn-call-for-testing.html) explaining what RTN is and how it works. For async closures, the lang team reached a preliminary consensus on the `async Fn` syntax, with the understanding that it will also include some "async type" syntax. This rationale was documented in [RFC #3710](https://github.com/rust-lang/rfcs/pull/3710), which is now open for feedback. The team held a [design meeting on Oct 23](https://hackmd.io/@rust-lang-team/ryxu3YLx1e) and @nikomatsakis will be updating the RFC with the conclusions.
+
+[sbp]: https://rust-lang.github.io/rust-project-goals/2024h2/async.html#resolve-the-send-bound-problem
+
+[RTN]: https://github.com/rust-lang/rfcs/pull/3654
+
+We have also been working towards a release of the `dynosaur` crate that enables dynamic dispatch for traits with async functions. This is intended as a transitionary step before we implement true dynamic dispatch. The next steps are to polish the implementation and issue a public call for testing.
+
+With respect to async drop experiments, @nikomatsakis began reviews. It is expected that reviews will continue for some time as this is a large PR.
+
+Finally, no progress has been made towards async WG reorganization. A meeting was scheduled but deferred. @tmandry is currently drafting an initial proposal.
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/116'><strong>Resolve the biggest blockers to Linux building on stable Rust</strong></a></div>
+    <div style="flex: initial;"><progress value="17" max="37"></progress>
+</div>
+</div>
+
+We have made significant progress on resolving blockers to Linux building on stable. Support for struct fields in the `offset_of!` macro has been stabilized. The final naming for the "derive-smart-pointer" feature has been decided as `#[derive(CoercePointee)]`; @dingxiangfei2009 prepared [PR #131284](https://github.com/rust-lang/rust/pull/131284) for the rename and is working on modifying the rust-for-linux repository to use the new name. Once that is complete, we will be able to stabilize. We decided to stabilize [support for references to statics in constants pointers-refs-to-static feature](https://github.com/rust-lang/rust/issues/128183) and are now awaiting a stabilization PR from @dingxiangfei2009.
```

Comment:
Hasn't the stabilization PR long landed?

Also, as a general note it'd be nice to give credit not only to the people filing the final stabilization PR, but also to the people doing the earlier implementation work. :)